### PR TITLE
Add links to github example config

### PR DESCRIPTION
**Pull request by realDragonium**
#### Changes:
- Add reference to hyprland.conf example file in master-tutorial
- Add reference to hyprland.conf example file in configuration-hyperland

#### Reason:
For the next noob trying this out. I found myself questioning what I needed to put into the the config file to change the default terminal. I think having a reference to this example config on github would have made me realise how to do it earlier, the example configuration setups on the wiki self were a bit too daunting, but in hindsight its fairly simple to do it.

### CODE REVIEW

File: pages/Configuring/Configuring-Hyprland.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 The config is located in `~/.config/hypr/hyprland.conf`.
 
 Hyprland will automatically generate an example config for you if
-you don't have one.
+you don't have one. you don't have one. Or you can find an example config [here](https://github.com/hyprwm/Hyprland/blob/main/example/hyprland.conf).
```
Comment (vaxerski) [inline pages/Configuring/Configuring-Hyprland.md:4]: ?

**Review by vaxerski — CHANGES_REQUESTED** [review]

File: pages/Configuring/Configuring-Hyprland.md (hunk 1 above)
Comment (realDragonium) [reply pages/Configuring/Configuring-Hyprland.md:4]:
```suggestion
you don't have one. Or you can find an example config [here](https://github.com/hyprwm/Hyprland/blob/main/example/hyprland.conf).
```

Good catch, where you talking about that? Or about something else?

Comment (vaxerski) [reply pages/Configuring/Configuring-Hyprland.md]: yes. Also, don't begin a sentence with "or". I'd rather "You can find the example config ..."

File: pages/Configuring/Configuring-Hyprland.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 The config is located in `~/.config/hypr/hyprland.conf`.
 
 Hyprland will automatically generate an example config for you if
-you don't have one.
+you don't have one. Or you can find an example config [here](https://github.com/hyprwm/Hyprland/blob/main/example/hyprland.conf).
```
Comment (realDragonium) [inline pages/Configuring/Configuring-Hyprland.md:4]:
```suggestion
you don't have one. You can find an example config [here](https://github.com/hyprwm/Hyprland/blob/main/example/hyprland.conf).
```

**Review by vaxerski — APPROVED** [review]
gangsta

**Pull request merged**